modules/migration-using-must-gather.adoc

```diff
@@ -19,13 +27,33 @@ The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migra
 ----
 $ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version}
 ----
++
+The data is saved in a local `must-gather` directory.
+
+.Viewing metrics data with Prometheus
```

Comment:
Remove this heading. You can make this into a single procedure. Also, if you put two procedures into a single module, you may be told during peer review to create a separate module for each procedure. Best to avoid, if possible.

Reply:
OK, the metrics_dump script collects the Prometheus metrics: https://github.com/konveyor/must-gather

Reply:
Is viewing the data with Prometheus an optional step? Also, what is its result? What does the user view the data for?
@apinnick

Reply:
Running must-gather with short command collects current state of all MTC resources and logs. Running it with the metrics_dump  collects Prometheus metrics for an entire day. That's why it takes so long.
It is an optional step. You could mark it as optional ("Optional: Do x" is the convention for OCP docs) but then you would need a subprocedure because the steps that follow are part of the same procedure.

Reply:
This changes my understanding of what is going on here. The Prometheus step is not about using Prometheus for viewing the data that has already been collected in the previous step. The Prometheus step actually involves collecting lots of additional data (does it include the data collected with the short must-gather command?) and then viewing it with Prometheus. But why isn't that expanded set of data collected regardless of Prometheus?
@apinnick

Reply:
Because it takes a long time to collect expanded data set. The most common complaint about must-gather is that it takes too long to run and gathers too much data. I am currently documenting ways to cut down on what must-gather collects.

Reply:
But then the procedure should be described differently: 
1. Collecting a restricted set of data with must-gather (without viewing it). 
2. Optional: Collecting an expanded set of data with Prometheus.
3. Optional: Viewing the expanded set of data with Prometheus.
Would this be correct?
But I don't quite understand the relation between must-gather and Prometheus. Why does running 'must-gather/metrics_dump' collect Prometheus metrics? 
@apinnick